### ISSUE TITLE
Update "git.thicc-thighs.de" to the latest Gitea version

### ISSUE THREAD

**Issue by NULLYUKI** [PrincessAkira/usefull-shit#2]
No idea in which repo I should open up this issue, so I just gonna open it up here.
You need to update your [Gitea](https://git.thicc-thighs.de/)
![grafik](https://github.com/PrincessAkira/egirl-dotfiles/assets/42326194/4fdd5fa9-6e5e-42df-9c5e-a3f9a119dd60)
The current latest version is [1.20.0-rc0](https://dl.gitea.com/gitea)

**Comment by NULLYUKI** [comment]
@PrincessAkira

**Comment by NULLYUKI** [comment]
Also a unblock EVERYWHERE would also still be great lol

**Comment by NULLYUKI** [comment]
@PrincessAkira

**Comment by HerXayah** [comment]
Its an RC version for a reason.
Wontfix ./

**Comment by NULLYUKI** [comment]
@PrincessAkira Seems like you are a bit blind. As there is a normal 1.20 version.
![grafik](https://github.com/PrincessAkira/egirl-dotfiles/assets/42326194/551b2047-b027-452e-b9b5-7a2258773ed3)
